lib/events.coffee

```diff
@@ -1,27 +1,32 @@
 _ = require('lodash')
-Mixpanel = require('mixpanel')
-Raven = require('raven')
+Analytics = require('analytics.node').core
+mixpanelIntegration = require('analytics.node').mixpanelIntegration
+ravenIntegration = require('analytics.node').sentryIntegration
 Promise = require('bluebird')
 resin = require('resin-sdk-preconfigured')
 packageJSON = require('../package.json')
+Analytics.addIntegration(ravenIntegration)
 
 exports.getLoggerInstance = _.memoize ->
```

Comment:
Can we rename this to `getAnalyticsInstance`? That seems clearer now, given the code here.